xarray/backends/cfgrib_.py

```diff
@@ -69,3 +73,72 @@ def get_encoding(self):
         dims = self.get_dimensions()
         encoding = {"unlimited_dims": {k for k, v in dims.items() if v is None}}
         return encoding
+
+
+def open_backend_dataset_cfgrib(
+    filename_or_obj,
+    *,
+    decode_cf=True,
+    mask_and_scale=True,
+    decode_times=None,
+    concat_characters=None,
+    decode_coords=None,
+    drop_variables=None,
+    use_cftime=None,
+    decode_timedelta=None,
+    lock=None,
+    indexpath="{path}.{short_hash}.idx",
+    filter_by_keys={},
+    read_keys=[],
+    encode_cf=("parameter", "time", "geography", "vertical"),
+    squeeze=True,
+    time_dims=("time", "step"),
+    **kwargs,
+):
+    if kwargs:
+        raise ValueError(
+            "The following keywords are not supported by the engine "
+            "and will be ignored: %r" % kwargs
+        )
```

Comment:
A simpler option is to just omit these lines and `**kwargs` entirely. Python will supply an error message like `TypeError: open_backend_dataset_cfgrib() got an unexpected keyword argument 'something_unexpected'`

Reply:
Yes, you are right! It doesn't make sense to keep these lines!